Structure_Engine/Create/Loads/LoadCombination.cs

```diff
@@ -57,20 +57,8 @@ public static LoadCombination LoadCombination(string name, int number, List<Load
                     factoredCases.Add(new Tuple<double, ICase>(factors[i], cases[i]));
             }
 
-            return LoadCombination(name, number, factoredCases);
-        }
-
-        /***************************************************/
-        /**** Public Methods - Deprecated               ****/
-        /***************************************************/
-
-        [Deprecated("3.1", "Replaced by Autogenerated property assignment method")]
-        public static LoadCombination LoadCombination(string name, int number, IEnumerable<Tuple<double, ICase>> factoredCases)
-        {
             return new LoadCombination { LoadCases = factoredCases.ToList(), Name = name, Number = number };
```

Comment:
Generally looks fine. details: Could get rid of the ToList() the factoredCases as it already is a list, aparat from that looks good.

Reply:
Resolved in [this commit](https://github.com/BHoM/BHoM_Engine/pull/2729/commits/4e57bc6a6e7af5e6fb404a5d605ebc2e659d6ce1)